server/app/assets/javascripts/dev_icons.ts

```diff
@@ -0,0 +1,11 @@
+window.addEventListener('load', () => {
+  Array.from(document.querySelectorAll('tr')).forEach((rowEl) => {
```

Comment:
suggestion (blocking): please document this file

Reply:
Done.